code/style.css

```diff
@@ -1,58 +1,214 @@
+}
+
+header h1 {
+  padding: 20px;
+  font-size: 36px;
 }
 
 p {
-  margin: 1px;
   font-weight: 300;
-  background-color: #fff;
+  background-color: grey;
+  font-size: 16px;
+  border-radius: 4px;
+  margin: 20px;  
 }
 
 .cards {
-  margin: 50px auto;
+  margin: auto;
   width: 90%;
-  height: 600px;
-  background: #ccc;
   display: flex;
   justify-content: space-around;
   align-items: center;
 }
 
+.allCards {
+  outline: none;
+  border: none;
+  border-radius: 5px;
+  box-shadow: 2px 4px 6px grey;
+  margin-bottom: 10px;
+}
+
+.allCards:hover {
+  background-color: #3e8e41;
+}
+
+.allCards:active {
+  background-color: #3e8e41;
+  box-shadow: 3px 4px 5px #666;
+  transform: translateY(4px);
+}
+
 .killer-deck {
-  width: 250px;
-  height: 400px;
-  background: #000;
+  width: 30%;
+  height: 530px;
+  background: rgb(212, 9, 9);
   text-align: center;
-  padding-top: 30px;
+  padding-top: 25px;
   border-radius: 16px;
 }
 
 .weapon-deck {
-  width: 250px;
-  height: 400px;
-  background: rgb(10, 53, 10);
+  width: 30%;
+  height: 520px;
+  background: rgb(56, 175, 211);
   text-align: center;
-  padding-top: 30px;
+  padding-top: 25px;
   border-radius: 16px;
 }
 
 .room-deck {
-  width: 250px;
-  height: 400px;
-  background: midnightblue;
-
+  width: 30%;
+  height: 520px;
+  background: rgb(68, 68, 216);
   text-align: center;
-  padding-top: 30px;
+  padding-top: 25px;
   border-radius: 16px;
 }
 
+img.questionmark  {
+  filter:sepia(100%);
+}
+
+/*BOUNCY LOADER*/
+
+.wrapper {
+  width: 100%;
+  text-align: center;
+  margin-top: 30px;
+}
+
+.bouncing-ball {
+  position: relative;
+  width: 4rem;
+  margin: 0 auto;
+  height: 12rem;
+}
+
+.ball {
+  width: 4rem;
+  height: 4rem;
+  border-radius: 50%;
+  background: black;
+  position: absolute;
+  top: 0;
+  animation: bounce 1s infinite;
+}
+
+@keyframes bounce {
+  0% {
+    top: 0;
+    animation-timing-function: ease-in;
+  }
+
+  34% {
+    height: 4rem;
+    width: 4rem;
+  }
+
+  35% {
+    top: 8rem;
+    height: 3rem;
+    width: 4.3rem;
+    animation-timing-function: ease-out;
+  }
+
+  45% {
+    height: 4rem;
+    width: 4rem;
+  }
+
+  90% {
+    top: 0;
+  }
+
+  100% {
+    top: 0;
+  }
+}
+
+.button-container {
+  text-align: center;
+}
+
 button {
-  padding: 8px 16px;
-  background: crimson;
+  padding: 16px 16px;
+  background: rgb(238, 228, 186);
   border: none;
-  font-size: 20px;
+  font-size: 26px;
+  font-weight: bolder;
   border-radius: 8px;
-  display: block;
-  margin: 0 auto;
+  margin: 0 50px;
+  margin-bottom: 50px;
+  font-family: 'Courier New', Courier, monospace;
+  min-width: 30%;
+  outline: none;
+  border: none;
+  border-radius: 10px;
+  box-shadow: 2px 4px 6px grey;
 }
+
+button:hover {background-color: #3e8e41}
+
+button:active {
+  background-color: #3e8e41;
+  box-shadow: 3px 5px 5px #666;
+  transform: translateY(4px);
+}
+
+h1.mystery {
+  text-align: center;
+  padding: 20px 200px;
+}
+
+/* MEDIA QUERY*/
+@media (max-width: 666px) {
```

Comment:
That's amazing how you did the media query for this project! Good job!

Reply:
I agree, this was very strong, well done @annahasselstrom